### DISCUSSION TITLE
How to run refetch after mutation success

### DISCUSSION THREAD

**Discussion by awixor** [reduxjs/redux-toolkit#4167] [Q&A]
I want to call `refetch` function after the mutation succeded:
```js
 const [joinTeam ] = useJoinTeamMutation();
 const { refetch } = useGetTeam(slugName);
```

I tried doing:

```js
leaveTeam().then((res: any) => {
     if (!res.error) {
          refetch();
     }
});
```

but this causes an error: `Cannot refetch a query that has not been started yet.`

**Comment by EskiMojo14** [comment]
have you looked at [automated refetching](https://redux-toolkit.js.org/rtk-query/usage/automated-refetching) using tags?
